### ISSUE TITLE
Maybe forward to origin now v3 is out

### ISSUE THREAD

**Issue by porsager** [Minigugus/postgres-deno#3]
This comes up first in google searches. Now that v3 is out with full Deno support, would it makes sense to forward back to porsager/postgres?

**Comment by Minigugus** [comment]
Obviously, that's what I planned do that do soon anyway :+1: However, there are now 2 packages about the `postgres` library on deno.land (`postgresql` and `postgresjs`), I don't know if that would be possible to merge both or at least redirect urls from mine to yours? I know deno.land wants immutability, but in that case, mine is just a clone now

Anyway, if that's not possible, I can still recycle this repository for a new postgres library, or even better: publish a final version with a single file containing a single line : `export * from 'https://deno.land/x/postgresjs'` :laughing:

**Comment by porsager** [comment]
Oh cool. Yeah i got this github repo on top on google searching for postgres and Deno. Hadn't even thought about the Deno package, i was just sad to see `postgres` was already taken 😊 

> publish a final version with a single file containing a single line : export * from 'https://deno.land/x/postgresjs' 😆

Lol, that would break with the idea about immutability pretty quickly 😝 

We could ask the Deno team on discord how they think best to handle it.
